src/layouts/menu/menu-list.ts

```diff
@@ -44,4 +40,31 @@ export default [
       },
     ],
   },
-] as MenuSection[];
+];
+
+export const adminMenu: MenuSection[] = [
+  {
+    type: 'main',
+    title: t('menu.admin'),
+    items: [
+      {
+        title: t('menu.community'),
+        href: '/contacts',
+        icon: ['fa', 'users'],
+        isActive: /^\/contacts.*/,
+      },
+      {
+        title: t('menu.callouts'),
+        href: '/admin/callouts',
+        icon: ['far', 'calendar-check'],
+        isActive: /^\/admin\/callouts.*/,
+      },
+      {
+        title: t('menu.notices'),
+        href: '/admin/notices',
+        icon: ['fa', 'bullhorn'],
+        isActive: /^\/admin\/notices.*/,
+      },
+    ],
+  },
+];
```

Comment:
Also, and kind of related to another thread on the admin dashboard:

I would really see the dashboard being at the top of the list, and potentially either the default page or at the root of the route (geddit?)

Reply:
Yeah I agree on all points. The "Dashboard" I have in the menu at the moment is actually just a link to the legacy app (as some functionality still only exists there). It needs a better name so we can add the actual dashboard to the menu!